core_lib_generator/core_lib_config_generate_yaml.py

```diff
@@ -60,10 +64,11 @@ def _get_solr_config():
 
 def _get_cache_config():
     print('Please fill out the requested Cache information.')
-    cache = generate_cache_template()
-    if 'env' in cache:
-        env.update(_get_env_variables(cache))
-    config['cache'] = cache['config']
+    cache_list = generate_cache_template()
+    for cache in cache_list:
+        if 'env' in cache:
+            env.update(_get_env_variables(cache))
+        config['cache'].append(cache['config'])

```

Comment:
config?

Reply:
we will have 2 sections in the returned dict `env` and `config` we will have to append the config section into the final list so for that reason